### ISSUE TITLE
Getting an error on loading checkpoints

### ISSUE THREAD

**Issue by Nerdyvedi** [xavysp/DexiNed#4]
I am getting the following error on running the model. 
`FailedPreconditionError (see above for traceback): Attempting to use uninitialized value Xpt/BatchNorm_34/beta
	 [[node Xpt/BatchNorm_34/beta/read (defined at /usr/local/lib/python3.5/dist-packages/tensorflow/contrib/framework/python/ops/variables.py:277)  = Identity[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Xpt/BatchNorm_34/beta)]]
`

I am using tensorflow version 1.12.3 

Should I initialize the weights after loading them ?

**Comment by Nerdyvedi** [comment]
Works with tensorflow 1.8
